File: MCA/PYTHON/Assignment2/Q2.py
n=int(input("Enter the range: "))

for i in range(n):
    print(" "*(n-i),end="")
    for j in range(1,i+2):
        print(j,end="")
    if(i!=0):
        for k in range(i,0,-1):
            print(k,end="")
    print()

File: MCA/JAVA/Assignment/D4/Q2.java
import java.util.Scanner;

public class Q2{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        int j=0;

        System.out.print("Enter Range: ");
        int n=sc.nextInt();

        for(int i=0;i<n;i++){
            for(int k=0;k<=i;k++){
                System.out.print(" "+j++);
            }
            System.out.println();
        }
    }
}
